[2022-04-20 14:31:34,248] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.transfer_xml scheduled__2022-04-19T17:00:00+00:00 [queued]>
[2022-04-20 14:31:34,289] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.transfer_xml scheduled__2022-04-19T17:00:00+00:00 [queued]>
[2022-04-20 14:31:34,290] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-20 14:31:34,290] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-04-20 14:31:34,290] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-20 14:31:34,315] {taskinstance.py:1264} INFO - Executing <Task(Postgres2File): transfer_xml> on 2022-04-19 17:00:00+00:00
[2022-04-20 14:31:34,320] {standard_task_runner.py:52} INFO - Started process 19202 to run task
[2022-04-20 14:31:34,334] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hook_dag', 'transfer_xml', 'scheduled__2022-04-19T17:00:00+00:00', '--job-id', '639', '--raw', '--subdir', 'DAGS_FOLDER/hook_dag.py', '--cfg-path', '/tmp/tmp2yh8ats9', '--error-file', '/tmp/tmpb82hyg1z']
[2022-04-20 14:31:34,338] {standard_task_runner.py:77} INFO - Job 639: Subtask transfer_xml
[2022-04-20 14:31:34,408] {logging_mixin.py:109} INFO - Running <TaskInstance: hook_dag.transfer_xml scheduled__2022-04-19T17:00:00+00:00 [running]> on host 22e04adb6bfa
[2022-04-20 14:31:34,598] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hook_dag
AIRFLOW_CTX_TASK_ID=transfer_xml
AIRFLOW_CTX_EXECUTION_DATE=2022-04-19T17:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-19T17:00:00+00:00
[2022-04-20 14:31:34,616] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-20 14:31:34,706] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=hook_dag, task_id=transfer_xml, execution_date=20220419T170000, start_date=20220420T143134, end_date=20220420T143134
[2022-04-20 14:31:34,739] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-20 14:31:34,913] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-20 14:31:34,922] {dbapi.py:225} INFO - Running statement: update list_flow_log
                set t_end=NOW(), status=2, comment = '*Task*: transfer_xml success'
                where id = 42;, parameters: None
[2022-04-20 14:31:34,924] {dbapi.py:233} INFO - Rows affected: 1
[2022-04-20 14:31:35,055] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
